Sources/GoogleMapsUtils/Clustering/StaticCluster/GMUStaticCluster1.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// TO-DO: Rename the class to `GMUStaticCluster` once the linking is done and remove the objective c class.
```

Comment:
Same comment about needing to rename `GMUCluster1` and `GMCluster1` in the implementation but that will probably be swept up in the Xcode refactoring job when you rename those classes so maybe you don't have to list them in the TODO.

Reply:
Kept this for my reference. So, when I will refactor the names, it will be easy for me to refactor. I will make sure, this are not missed.